client/src/modules/employees/registry/search.modal.html

```diff
@@ -66,58 +66,46 @@
             </label>
           </div>
 
-          <div class="form-group">
-            <label class="control-label" translate>
-              FORM.LABELS.GRADE
-            </label>
-
+          <bh-grade-select
+            grade-uuid="ModalCtrl.searchQueries.grade_id"
```

Comment:
Why is this `grade-uuid`, but it takes in a `grade_id`?  Is the grade a uuid or an id?

Reply:
Because in API in server part all sql request Make the Jointure like this
````
JOIN grade ON employee.grade_id = grade.uuid
````

Reply:
O.o.  Okay, we should probably fix this.  It looks like it is stored in the database as a `uuid`, but the column is called `id` for some reason.